packages/http/src/Client.ts

```diff
@@ -34,22 +34,29 @@ interface BlockFromHeightOrHashFn {
   (heightOrHash: number | string): Block
 }
 
+interface Options {
+  retry: number | false
+}
+
 export default class Client {
   public network!: Network
-
+  public retry!: number | false
```

Comment:
I'm wondering if instead of false, we just keep it a number and you would pass 0 for no retries?

Reply:
yeah that's probably cleaner, I actually saw axios or another library had that same pattern somewhere, good call 👍